docs/02-development-environments/01-devcontainer.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Developing Discourse using a 'Dev Container'
+short_title: Dev Container
+id: dev-container
+
+---
+
+[Dev Containers](https://containers.dev/) is an open standard for configuring a development environment inside a container. This almost entirely eliminates the need to install/configure Discourse-specific tools/dependencies on your local machine, and makes it very easy to keep up-to-date as Discourse evolves over time.
+
+Dev Containers can be used in a number of different IDEs, or directly using their reference CLI. This guide will describe the setup process for VSCode.
+
+## Getting started
+
+1. [Download and install](https://code.visualstudio.com/) VSCode
+
+1. Install the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) in VSCode
+
+1. Clone the Discourse repository onto your machine
+   ```
+   git clone https://github.com/discourse/discourse
+   ```
+
+1. In VSCode, use `File` -> `Open Folder`, then choose the Discourse directory
+
+1. Open the folder in its Dev Container. This can be done via the popup prompt, or by opening the command pallette (Cmd/Ctrl + Shift + P) and searching for "Open folder in container..."
```

Comment:
```suggestion
1. Open the folder in its Dev Container. This can be done via the popup prompt, or by opening the command palette (<kbd>Cmd/Ctrl + Shift + P</kbd>) and searching for "Open folder in container..."
```